crowd_anki/representation/note_model.py

```diff
@@ -33,7 +33,7 @@ def check_semantically_identical(first_model, second_model):
 
         return True
 
-    def save_to_collection(self, collection: Collection):
+    def save_to_collection(self, collection: Collection, should_save: bool):
```

Comment:
https://github.com/Stvad/CrowdAnki/pull/72/files#r446706208

Reply:
See my other comment on `save_note_model` not being replaced. Also this example further shows the need to pass it into here, as I need to call *part* of this function in order to keep everything else happy and working. I did elaborate in one of my top level comments in the PR:

> Note Models I had to pass the bool into the save_to_collection function so that I could do part of it, to update the anki_dict, as the id is used in the notes importer.